data/json/npcs/talk_tags.json

```diff
@@ -988,7 +1287,14 @@
       "Hold up, gotta plug this hole in me.",
       "Watch my back while I stitch my arm back on <name_g>.",
       "Gotta bandage this or I'll bleed out.  Give me a sec.",
-      "I ain't got time to ble-wait, that's a lot of blood.  Give me a second while I patch this."
+      "I ain't got time to ble-wait, that's a lot of blood.  Give me a second while I patch this.",
+      "Crap, sprung a leak.  Keep me safe while I patch this up.",
+      "That's a nasty wound I got.  Gonna need to cross my fingers and hope this works.",
+      "Yeah, I'm a little banged up here.  Give me a sec to take care of this.",
+      "<swear!>, that stings.  I hopefully have something for it.",
```

Comment:
`Hopefully I have`

Reply:
That sounds a bit better, will change this.